src/main/java/com/exasol/releasedroid/adapter/github/GitHubReleaseMaker.java

```diff
@@ -48,11 +49,22 @@ private String createGitHubRelease(final Repository repository) {
 
     private GitHubRelease createReleaseModel(final Repository repository, final String version) {
         final ReleaseLetter releaseLetter = repository.getReleaseLetter(version);
+        String header = releaseLetter.getHeader().orElse("");
+        if (header.isEmpty()) {
+            throw new IllegalStateException(ExaError.messageBuilder("E-RD-GH-28") //
+                    .message("Release header must not be empty.") //
+                    .mitigation("Please provide release letter with non-empty header.") //
+                    .toString());
+        }
```

Comment:
There is also a method `Optional.orElseThrow()`. But this also catches the case when the header is `""` 👍